vispy/visuals/text/_sdf_cpu.pyx

```diff
@@ -7,6 +7,8 @@ cimport numpy as np
 from libc.math cimport sqrt
 cimport cython
 
+np.import_array()
```

Comment:
This is also from ruff fixes?

Reply:
Actually no, this is from a warning. I thought it was related to numpy 2.0, but now that I look maybe not.

See this example in the [Cython docs](https://cython.readthedocs.io/en/latest/src/tutorial/numpy.html#adding-types):

```
# "cimport" is used to import special compile-time information
# about the numpy module (this is stored in a file numpy.pxd which is
# distributed with Numpy).
# Here we've used the name "cnp" to make it easier to understand what
# comes from the cimported module and what comes from the imported module,
# however you can use the same name for both if you wish.
cimport numpy as cnp

# It's necessary to call "import_array" if you use any part of the
# numpy PyArray_* API. From Cython 3, accessing attributes like
# ".shape" on a typed Numpy array use this API. Therefore we recommend
# always calling "import_array" whenever you "cimport numpy"
cnp.import_array()
```